Framework/Algorithms/test/GetEiV1Test.h

```diff
@@ -170,17 +172,15 @@ class GetEiV1Test : public CxxTest::TestSuite {
     const double peakOneCentre(6493.0), sigmaSqOne(150. * 150.),
         peakTwoCentre(10625.), sigmaSqTwo(25. * 25.);
     const double peakOneHeight(3000.), peakTwoHeight(1000.);
-    for (int i = 0; i <= numBins; ++i) {
-      const double xValue = 5.0 + 5.5 * i;
-      if (includePeaks && i < numBins) {
+    for (int i = 0; i < numBins; ++i) {
+      if (includePeaks) {
         testWS->dataY(0)[i] =
             peakOneHeight *
-            exp(-0.5 * pow(xValue - peakOneCentre, 2.) / sigmaSqOne);
+            exp(-0.5 * pow(xdata[i] - peakOneCentre, 2.) / sigmaSqOne);
```

Comment:
Nevermind, making the exponent integral looks like outdated advice. The assembly generated for  `x*x` and std::pow(x,2.) is identical.

https://godbolt.org/g/CuL895